src/app/signing-video/sign-language.component.scss

```diff
@@ -0,0 +1,30 @@
+// this should not be necessary (w-100 on youtube-player should be enough),
+// but there is a bug with responsiveness in the youtube player component right now:
+// https://github.com/angular/components/issues/28829
+youtube-player {
+  position: relative;
+  padding-top: 56.25%; // 16:9 ratio
+}
+
+:host ::ng-deep youtube-player > *, {
```

Comment:
`::ng.deep` kannte ich bisher nicht, die Doku rät dazu das nicht mehr zu benutzen. Spricht irgendwas konkretes für diesen Fall?

https://angular.dev/guide/components/styling#ng-deep

Reply:
Soweit ich weiß gibt es aktuell keinen besseren Weg, Styles auf Child-Elemente einer Komponente anzuwenden, die wie hier aus einer anderen Bibliothek kommen. Bis dahin bleibt einem nicht viel übrig als weiterhin ng-deep zu benutzen.

Hier der Abschnitt aus der Doku zu Angular v17: https://v17.angular.io/guide/component-styles#deprecated-deep--and-ng-deep

Hat sich in diesem Fall aber auch erledigt, da ja jetzt nicht mehr bei YouTube gehostet wird.